Added detail slider + sketch-prompt slider

matooo3 opened the pull request:
## Resolves #27 and #28 

## short Description:
Added working sliders, changing values in js script, so we can use it later to set values for the python script and backend

## Changes:
- added Slidebar for Details from 0% to 100%
   - if the slidebar value is changed, a value in the js script is changed (detailValue)
- same for Slidebar for Sketch-Prompt weigths
   - changed value: spValue
- changed slidebar style according to the current Application style (red-themed)
---
Please review and let me know, if you find any bugs :)

matooo3 committed:
added detail slider + sketch-prompt slider

Added working sliders, changing values in js script, so we can use it later to set values for the python script and backend

matooo3 committed:
fixed hover bug

fixed bug (hover scale 1.2 was not working)

philipfsr reviewed (approved):
Tested your code locally and works fine. Could not find any issues.

One thing that could be done is a description for the sliders when hovering over them (like how we did with the brush sizes for example).
But for now I think you can merge it

matooo3 committed:
fixes

fixed 2 values for sketch/prompt, side description and hover description

matooo3 commented:
> Tested your code locally and works fine. Could not find any issues.
> 
> One thing that could be done is a description for the sliders when hovering over them (like how we did with the brush sizes for example). But for now I think you can merge it

Thanks for your review! I added following changes:
- added hover descriptions of each slider
- added side description of each slider
- added sValue and pValue instead of inly spValue (extra value for prompt and sketch, for py file compatibility)
